site/getenvoy_installer.sh

```diff
@@ -0,0 +1,401 @@
+#!/bin/sh
```

Comment:
This is a copy. After updating the _redirects, I'll go back and delete the old one